blog/2023-10-25-image-editor.md

```diff
@@ -125,16 +137,15 @@ corners disappear.
 So, to cover up these checkered corners, we will need to scale the image. If we
 cover up the larger corner, the smaller corner will get covered up
 automatically, so our code takes the form of
-`scale(Math.max(scalingFactor1, scalingFactor2))`.  
-These two scaling factors are calculated very similarly, so we’ll only focus on
-calculating only one of them in this tutorial (a full solution is given in the
-conclusion too, however).
+`scale(Math.max(scalingFactor1, scalingFactor2))`. These two scaling factors are
+calculated very similarly, so we’ll only focus on calculating only one of them
+in this tutorial (although the full solution is given in the conclusion).
 
 In the images below, the <span style={{ color: `rgb(127, 194, 65)` }}>green
 rectangle</span> represents the desired dimensions of our image after it’s
-scaled. Scaling operation is defined in such a way that our scaling factor is
-`H/h`. We already know `h` (it’s the height of our image!), so we want to find
-`H`.
+scaled. Luckily for us, the scaling function is defined in such a way that our
```

Comment:
Came up with a better alternative than both of our versions, tell me what you think:

Our scaling function (and, hopefully, yours - in general that's how scaling is defined in linear algebra) is defined in such a way that if we have the image of height `h`, and we want to scale it up to height `H`, we want to execute `.scale(H/h)`. We already know `h`, the height of our image, and now we want to find `H` - all of the further steps in this tutorial will be automatic - we know all the angles in this image, we know the image's width and height, and we want to find `H`.

Reply:
I like this a lot :) Just added it and changed a word ere and there so it flows a little nicer but definitely a big improvement.